### DISCUSSION TITLE
Linux port?

### DISCUSSION THREAD

**Discussion by nabelekt** [microsoft/terminal#17512] [Ideas]
Terminal is the best CLI app I've used. Any plans to port it to Linux? There's nothing out there for Ubuntu Desktop that looks quite as good. I haven't tried it yet, but https://extraterm.org/ may be the closest.

**Comment by j4james** [comment]
I'm afraid that's not at all likely to happen. This is from the FAQ:

> [What about porting the Windows Terminal to MacOS or Linux?](https://github.com/microsoft/terminal/wiki/Frequently-Asked-Questions-(FAQ)#what-about-porting-the-windows-terminal-to-macos-or-linux)
>
> Windows Terminal requires a good number of Windows-specific technologies. We unfortunately won't be supporting it on Mac or anywhere else any time soon.

**Reply by nabelekt** [reply]
Bummer. Thanks for the info.
